src/components/handbook-navigation/index.js

```diff
@@ -1,19 +1,42 @@
-import React from "react";
+import React, { useState } from "react";
 import { HiOutlineChevronLeft } from "react-icons/hi";
 import { Link } from "gatsby";
 import TOCWrapper from "./toc.style";
+import { IoMdClose, IoIosArrowDropdownCircle } from "react-icons/io";
 
 const TOC = () => {
+  const [expand, setExpand] = useState(false);
   return (
     <TOCWrapper>
       <div className="go-back">
         <Link to="/community/handbook">
           <HiOutlineChevronLeft />
           <h4>Table of Content</h4>
         </Link>
+        <div className="toc-toggle-btn">
+          {expand ? (
+            <IoMdClose
+              className="mobile-menu-icon open"
+              style={{ width: "1.5rem", height: "1.5rem", cursor:"pointer" }}
```

Comment:
Please avoid inline-styles. Use `className` instead and add the CSS in the stylesheets.
It will help avoid duplicity, as you can see from line: 30.